backend/api/Areas/Reports/Mapping/Project/ProjectMap.cs

```diff
@@ -34,6 +35,8 @@ public void Register(TypeAdapterConfig config)
                 .Map(dest => dest.Description, src => src.Description)
                 .Map(dest => dest.ReportedFiscalYear, src => src.ReportedFiscalYear)
                 .Map(dest => dest.ActualFiscalYear, src => src.ActualFiscalYear)
+                .Map(dest => dest.ReportedFiscalYearString, src => src.ReportedFiscalYear.FiscalYear())
```

Comment:
I needed to add 2 new fields to the mapping, so now the report will show both the fiscal year as well as the formatted fiscal year "string" for both the reported and actual fiscal year.